test/sql/attach/attach_fsspec.test

```diff
@@ -0,0 +1,13 @@
+# name: test/sql/attach/attach_fsspec.test
```

Comment:
@Mytherin I wasn't able to get these tests to run locally unfortunately, not sure if there's a trick to it? Some list they need to be added to?

Reply:
They are automatically added - but you need to run the `unittest` program from the root directory, e.g.:

```
build/debug/test/unittest test/sql/attach/attach_fsspec.test
```